MultigridProjector/Logic/MultigridProjection.cs

```diff
@@ -37,7 +38,7 @@ namespace MultigridProjector.Logic
     // FIXME: Refactor this class
     public class MultigridProjection
     {
-        private const int UpdateCooldownTime = 2000; // ms
+        public static int UpdateCooldownTime { get; internal set; }  = 2000; // ms
```

Comment:
I suggest adding a publicly writable `bool UpdateContinuously` flag instead. Then the getter of `UpdateCooldownTime` can return 100 instead of its current value if the flag is enabled. The highlighter should enable that flag, so it does not need to store an "old" value.